app/src/androidTest/java/org/simple/clinic/MigrationAndroidTest.kt

```diff
@@ -1,29 +1,25 @@
 package org.simple.clinic
```

Comment:
Can you change usages of backticks in SQL statements to double quotes please?

Reply:
Since this class has usages of both backticks and double quotes, with backticks being the majority, let's keep this for now until we decide on one standard. I'll add this to my discuss-once-pratul-is-back TODOs.

Reply:
@vinaysshenoy changing them to double quotes